Makefile.am

```diff
@@ -5325,6 +5323,11 @@ endif
 
 # ------------------------------------------------------------------------------
 if ENABLE_MACHINED
+
+dist_systemunit_DATA += \
+	units/var-lib-machines.mount \
+	units/var-lib-machines.automount
```

Comment:
this part is simply wrong, nspawn uses this too, see other comment.

In fact systemd-nspawn@.service should probably get an explicit RequiresMountsFor= for the container dir it operates on